src/headless/main.cpp

```diff
@@ -72,9 +102,27 @@ static bool run_headless(Dataloader::path_vector_t const& roots, bool run_tests)
 		CountryInstanceManager const& country_instance_manager =
 			game_manager.get_instance_manager()->get_country_instance_manager();
 
-		print_ranking_list("Great Powers", country_instance_manager.get_great_powers());
+		std::vector<CountryInstance*> const& great_powers = country_instance_manager.get_great_powers();
+		print_ranking_list("Great Powers", great_powers);
 		print_ranking_list("Secondary Powers", country_instance_manager.get_secondary_powers());
 		print_ranking_list("All countries", country_instance_manager.get_total_ranking());
+
+		Logger::info("===== RGO test... =====");
+		for (int i = 0; i < 3; ++i) {
+			CountryInstance const* const great_power = great_powers[i];
+			if(great_power == nullptr) {
+				Logger::warning("Great power ",std::to_string(i), " is null.");
```

Comment:
No need to use `std::to_string` inside `Logger` messages, `int`s are converted automatically.